src/endpoints/base_endpoint_registry.cpp

```diff
@@ -243,4 +244,18 @@ namespace ccf
       return ApiResult::InternalError;
     }
   }
+
+  ApiResult BaseEndpointRegistry::get_untrusted_host_time_v1(::timespec& time)
+  {
+    const std::chrono::microseconds now_us = enclave::get_enclave_time();
+
+    constexpr auto us_per_s = 1'000'000;
+    time.tv_sec = now_us.count() / us_per_s;
+    time.tv_nsec = (now_us.count() % us_per_s) * 1'000;
+
+    LOG_INFO_FMT(
+      "{} -> {{ {}, {} }}", now_us.count(), time.tv_sec, time.tv_nsec);
```

Comment:
this should probably be at a lower log level

Reply:
This was a debugging line, has now been removed.